python/tk_multi_launchapp/software_entity_launcher.py

```diff
@@ -384,6 +384,16 @@ def _scan_for_software_and_register(self, engine_str, dcc_versions, dcc_products
                     )
                     continue
 
+            # We need to check to see if the engine instance associated with
+            # the launch command is something we've been configured to skip.
+            if launch_engine_str in self._tk_app.get_setting("skip_engine_instances"):
+                self._tk_app.log_debug(
```

Comment:
I think it's safe to call `self._tk_app.logger.debug` since you are guaranteed to have a 0.18 core. Plus, you will be able to replace ` % (launch_engine_str, software_version)` with `, launch_engine_str, software_version`